test-apps/display-performance-test-app/package.json

```diff
@@ -73,8 +73,8 @@
     "npm-run-all": "^4.1.5",
     "null-loader": "^0.1.1",
     "rimraf": "^3.0.2",
-    "react": "^16.8.0",
-    "typescript": "~4.3.0",
+"react": "^17.0.0",
```

Comment:
looks like display team should remove unneeded dependencies from our test apps...

Reply:
Would like me to remove it in this PR?

Reply:
If it's no trouble, please go ahead. Thanks.